SharePointFramework/shared-library/src/components/ProjectLogo/types.ts

```diff
@@ -14,6 +14,21 @@ export interface IProjectLogoProps {
    */
   url: string
 
+  /**
+   * Size of the project logo
+   */
+  size?: string
+
+  /**
+   * Size of the projectLogo
```

Comment:
type is not "Size of the projectLogo". Could this property be renamed, or documented so it's understandable?

Reply:
Fikset